src/main/java/GameMonitor.java

```diff
@@ -1,10 +1,32 @@
-import com.fasterxml.jackson.databind.JsonNode;
-
 public class GameMonitor {
-    public void buttonPress(JsonNode clickPayload) {
-        String button = clickPayload.get("button").asText();
-        String actionType = clickPayload.get("actionType").asText();
 
-        System.out.println(String.format("Button %s sent the action %s", button, actionType));
+    Players firstPlayer;
+    Players secondPlayer;
+    Game spiel;
+
+    public GameMonitor () {
+        this.firstPlayer = new Players();
+        this.secondPlayer = new Players();
+        this.spiel = new Game(firstPlayer, secondPlayer);
+    }
+
+    public void buttonPress(Message message) {
+
+         if (message.getButtonId().equals("1")) {
+             spiel.updateScoreOfPlayer(firstPlayer, message.getActionType());
+             if (spiel.hasSomeoneOfThePlayersElevenPointsYet()) {
```

Comment:
The code here from line 17 to 19 is exactly the same as line 23 to 25. As we spoke about earlier we should avoid such duplication #DRY